SoftwareDesignDocument..txt

```diff
@@ -26,10 +26,46 @@ Code:
 				                    // Dimensions of the tank
 				                    result = set()
 				                    return result
-        
- 
+Req 1.0.1
+	Rules for tank movement.
+Code:
+	void tankMovement()
+	     	// Speed is just for an example value
+		// Will move with same controls as character
+		int tank_velocity = 2;
+		//Block height 1 more than current level of tank automatically move on top of block
+Req 1.0.2
+	Despawn the tank after 5 collisions with projectile block.
```

Comment:
I think it would be easier if we just used the hp system from vanilla minecraft.

Reply:
That's a good idea, but I don't think the craft that we are using doesn't have a health function we could reuse. If I remember Minecraft has 10 hearts so I was just doing 2 hearts of damage each time if there is an health function I missed.

Reply:
Yeah that sounds good, I just checked and this version doesn't have the heart system like I thought it did.